src/common/loggers/csv_logger.cc

```diff
@@ -21,6 +23,9 @@ CsvLogger::CsvLogger(size_t log_id, const std::string& radio_name) {
   if (log_id >= kAllLogs) {
     AGORA_LOG_ERROR("Invalid log id %zu in CsvLogger\n", log_id);
   } else {
+    if (std::filesystem::is_directory("files/log") == false) {
```

Comment:
spdlog creates the directory automatically if not exists. No need to create the directory manually.